src/styles/blocks/active.scss

```diff
@@ -0,0 +1,16 @@
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
```

Comment:
Why don't you use nesting with `&` operator?

Reply:
I forgot. 
I will change it, thanks!